front/app/feed.xml/route.ts

```diff
@@ -13,23 +22,29 @@ export async function GET() {
     ttl: 6 * 60, // 6 hours
   });
 
-  const comics = await getAllMetadata();
-  const orderedComics = useComicMetadata(comics, "upload");
-  orderedComics.forEach((comic) => {
-    const firstImageLink = comic.series.comics[0].image_urls[0];
-    const firstImage = comic.images[firstImageLink];
-    const articleImageLink = getImageUrl(firstImage);
-    const description = firstImage.ocr || "A Comic";
+  const metadata = await getAllMetadata();
+  let comics : {comic: Comic, series: Metadata}[] = []
+  metadata.forEach((series) => {
+    Object.values(series.series.comics).forEach((comic) => {
+      comics.push({comic, series})
+    })
+  });
```

Comment:
```suggestion
  const comics = metadata.flatmap(meta => {
      const {series} = meta
      return series.comics.map(comic => ({comic, series}))
  });
```

You are here trying to _unwrap_ this 2D list like data structure into a 1D list.

For this you can use `map` to change the individual values, and `flatmap` to convert this list of lists to a simple list.

Reply:
i have switched to this as well as switched the naming up to be `comicsWithMetadata`/`comicWithMetadata` (and made that a type instead of having to put the whole struct everywhere)